habitat-baselines/habitat_baselines/README.md

```diff
@@ -4,8 +4,8 @@ baselines
 
 The `habitat_baselines` sub-package is NOT included upon installation by default. To install `habitat_baselines`, use the following command instead:
 ```bash
-pip install -r requirements.txt
-python setup.py develop --all
```

Comment:
Regarding, develop. That was expected that develop installs pytest lib required for testing during development. Will this option work as well?

Reply:
The expectation was that `python setup.py develop --all` would install both habitat-lab and habitat-baselines, but now that the two packages are separate, I don't think it makes sense anymore.
It would be weird to have a command on habitat-lab that installs habitat-baselines.

Regarding the pytest lib installation, on this line : https://github.com/vincentpierre/habitat-lab/blob/split_setup/habitat-lab/setup.py#L55 the pytest libraries are a still here, the libraries would be installed the same way (but only habitat-lab would be installed, not habitat-baselines)

Reply:
Yes, so `develop` options still works for `habitat-lab` and `habitat-baselines` separately.